src/Halcyon/MemoryRepository.php

```diff
@@ -29,7 +29,9 @@ public function get($key, $default = null)
             return $this->many($key);
         }
 
-        if($value = $this->getFromMemoryCache($key)) {
+        $value = $this->getFromMemoryCache($key);
+
+        if(isset($value)) {
```

Comment:
I'd prefer that you use `!is_null($value)` instead for this, since that is what you are actually testing for, the fact that `isset()` returns `false` for `null` is not immediately obvious unless you're already aware of that fact so for readability I would prefer the `!is_null()` check instead. Thoughts?

Reply:
I agree, I was actually checking out whether or not `isset()` returns true for `null` myself. Better to just use `!is_null()` in the first case.